src/v3/src/util/getLanguageDirection.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2023-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+const rtlLanguageCodes = new Set([
+  'ar', // Arabic
+  'arc', // Aramaic
+  'arz', // Egyptian Arabic
+  'ckb', // Kurdish (Sorani)
+  'dv', // Divehi
+  'fa', // Persian
+  'ha', // Hausa
+  'he', // Hebrew
+  'khw', // Khowar
+  'ks', // Kashmiri
+  'ps', // Pashto
+  'sd', // Sindhi
+  'ur', // Urdu
+  'uz', // Uzbeki Afghanistan
+  'yi', // Yiddish
+]);
+
+// ltr is default so mark as undefined when ltr
+type LanguageDirection = 'rtl' | undefined;
```

Comment:
IMO we should either name this `LanguageIsRTL` or something similar or strictly make it a string union of `'rtl' | 'ltr'`

Reply:
Yea I was trying to make it so that when the language is `ltr` it doesnt add the property to the dom, it doesnt hurt anything having it there I just wanted to avoid unnecessary noise in snapshots and in the html adding `ltr` when it is set that way by default. But I agree it is confusing only having it as `rtl`